cmd/containerd/command/service_windows.go

```diff
@@ -337,6 +339,9 @@ func initPanicFile(path string) error {
 	// and replace it.
 	if st.Size() > 0 {
 		panicFile.Close()
+
+		logFileInChunks(path)
```

Comment:
Missed this in first review, but why have the func return an error if we don't check? We do log in a couple places but there's other where we return the err directly also